### PR TITLE
Add smartthings-connector extension

### PR DESCRIPTION
**Pull request by 4IngoJ**
## Description

An extension that makes it possible to control your Samsung Smartthings IOT devices through Raycast. 

## Screencast

🎦 https://drive.google.com/file/d/1O5GbORuYQITb7jNhM4mFTR-e79nBy9dc/view?usp=sharing

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/smartthings-connector/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Smartthings Connector Changelog
+
+## [Initial Version] - 2024-09-03
```
Comment (pernielsentikaer) [inline extensions/smartthings-connector/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - {PR_MERGE_DATE}
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- What is `catworkx academy symbols.afdesign`

- Could you add at least one [screenshot](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots) of the extension

- Remove `command-icon.png` since it's not in use




I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by 4IngoJ** [comment]
Hi,

Thanks for the feedback.

* The `catworkx academy symbols.afdesign` file accidentally sneaked into the project - deleted it
* I have deleted the command-icon.png
* Here are some Screenshots of the extension.

2e9a40e70715eb532eae4725a4dbe70d6659cd6eb7fb0f31c77e1a093c97ebcf
6a23e6d83d4b3e0f987471fda88cf48771a9c39d777f7a2101d77384c1f4eec1
ce5cbece3e4fd318d5f2eb38a816d658519082d3999c9fb329a350c7e0fcb029
d988053587201d984943d095f65548ddf94b8f4b0cbf6e080b9f91d249a0c76e
59c9657da0e75474a3c016b2582c627548e1a5e3b988be2a9e9fbf1bb96a43
![eae36ca41a13601c5a2858927c878052c18ee28be3310db4e0712e35217dfcfd
a9](https://github.com/user-attachments/assets/88f668ac-6df0-47e0-8ac3-510fb5055c81)

**Comment by pernielsentikaer** [comment]
Looks much better 🔥

Could you move the screenshots to a folder called `metadata` 🔥

**Comment by pernielsentikaer** [comment]
Did you push the changes? 🙂

**Comment by 4IngoJ** [comment]
> Did you push the changes? 🙂

Yeah - forgot it, but now i have 👍

**Comment by 4IngoJ** [comment]
Hi @pernielsentikaer 
I added other screenshots on an Retina Display as said in the failed check, but now pushing again (npm run publish) fails:

image

Could this be an error on your side?

**Comment by 4IngoJ** [comment]
> Hi @pernielsentikaer I added other screenshots on an Retina Display as said in the failed check, but now pushing again (npm run publish) fails:
> 
> image
> Could this be an error on your side?

@pernielsentikaer 
Was a Github buffer Error, publishing worked again, please check again.

File: extensions/smartthings-connector/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+module.exports = {
```
Comment (pernielsentikaer) [inline extensions/smartthings-connector/.eslintrc.js:1]: Are you using an old version? New plugins should have `.eslintrc.json` with this

```
{
  "root": true,
  "extends": ["@raycast"]
}
```

Comment (4IngoJ) [reply extensions/smartthings-connector/.eslintrc.js:1]: > Are you using an old version? New plugins should have `.eslintrc.json` with this
> 
> ```
> {
>   "root": true,
>   "extends": ["@raycast"]
> }
> ```

Hi,
This was the best running version for my use case. Is it necessary to change the lint config?

Comment (pernielsentikaer) [reply extensions/smartthings-connector/.eslintrc.js:1]: What was not working in a newer version of Raycast?

Comment (4IngoJ) [reply extensions/smartthings-connector/.eslintrc.js:1]: Changed the version and worked now just fine :)
image

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**